### PR TITLE
Fix broken link to Ritchie FAQs

### LINKED ISSUE #576: Fix broken link CONTRIBUTING.md
The community guidelines link is broken in repositories.  
Currently pointing to [Ritchie community guidelines](https://docs.ritchiecli.io/community), link to the [FAQs](https://docs.ritchiecli.io/faq) seems to be a good option.

I'd like to make the above changes if they seem correct. 
(Other repositories also have the same issue, should I send PR for each of them?)

### PR DESCRIPTION
**Pull request by Harirai**
**- What I did**
I removed the community guidelines broken link and replaced it with an appropriate link pointing to [FAQs](https://docs.ritchiecli.io/faq).
Closes #576
**- How to verify it**
By clicking on the link.
**- Description for the changelog**
Replaced broken link with [FAQs](https://docs.ritchiecli.io/faq).

### CODE REVIEW

**Comment by GuillaumeFalourd** [comment]
You just have to sign off the DCO @Harirai and it will be all good 👍

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ help. If a reviewer realizes you have based your work on the wrong branch, we'll
 let you know so that you can rebase it.
 
 >**Note**: To contribute code to Ritchie projects, see the
-[Ritchie community guidelines](https://docs.ritchiecli.io/community) as well as the 
+[Ritchie FAQs](https://docs.ritchiecli.io/faq) as well as the
```
Comment (GuillaumeFalourd) [inline CONTRIBUTING.md:12]: As it's the "community" guidelines, what do you think to directly use this link: https://docs.ritchiecli.io/faq#community ?

Comment (Harirai) [reply CONTRIBUTING.md:12]: Yeah sure that will be great!

**Review by lucasdittrichzup — APPROVED** [review]

**Review by GuillaumeFalourd — APPROVED** [review]

**Comment by GuillaumeFalourd** [comment]
Thank you for contributing @Harirai  👍

**Pull request merged**